gulpfile.js

```diff
@@ -0,0 +1,101 @@
+var gulp = require('gulp');
+var path = require('path');
+var oppiaGae = require('oppia-gulp');
+var concatCSS = require('gulp-concat');
+var minifyCSS = require('gulp-minify-css');
+var ifElse = require('gulp-if-else');
+var argv = require('yargs')
+    .usage('Usage: $0 --appengine [Path to app engine]' +
+      '--clear_datastore [clear all data storage]' +
+      '--enable_sendmail [send emails]' +
+      '--dev_mode [variable that shows if is in dev_mode]')
+    .demand(['appengine']).argv;
+var manifest = require('./manifest.json');
+
+var serveCmd = argv.appengine;
+var params = {
+  admin_host: '0.0.0.0',
+  admin_port: 8000,
+  host: '0.0.0.0',
+  port: 8181
+};
+
+var is_dev = argv.dev_mode == 'True';
```

Comment:
Also, here and below, standardize naming conventions: use camelCase and not underscores